flashcards/backend/decks/models.py

```diff
@@ -100,15 +100,15 @@ class Tag(models.Model):
     """Model for representing tags for decks
     :param name: Tag name
     :type name: str
-    :param deck: Foreign Key for Deck model
-    :type deck: int
+    :param decks: Foreign Key for Deck model
+    :type decks: int
     :param flashcard: Foreign Key for Flashcard model
     :type flashcard: int
     """
 
     name = models.CharField(max_length=24)
-    deck = models.ForeignKey("Deck", on_delete=models.DO_NOTHING)
-    flashcard = models.ForeignKey("Flashcard", on_delete=models.DO_NOTHING)
+    decks = models.ManyToManyField("Deck", related_name="tags")
+    flashcard = models.ForeignKey("Flashcard", on_delete=models.DO_NOTHING, related_name="tags")
```

Comment:
The flashcard should also be a MM relation. while we are here I'm wondering, if adding `blank=True` to those fields to make them not required while creating a tag.